### ISSUE TITLE
Find All and Replace don't work

### ISSUE THREAD

**Issue by tylerlong** [coderaiser/cm-searchbox#1]
I created a demo project for cm-searchbox: https://github.com/tylerlong/cm-search-demo

`Find All` and `Replace` don't work, while `Find` and `Replace All` do work. 

Please have a try and confirm. 

Thank you!

**Comment by coderaiser** [comment]
`Replace` works properly in [cm-search-demo](https://github.com/tylerlong/cm-search-demo), can't reproduce it.
`Replace All` should just select words position in scroll bar, multiple cursors not work for now.

Feel free to make pull request.

**Comment by tylerlong** [comment]
I got the following error when `Replace`:

```
searchbox.js:296 Uncaught TypeError: Cannot read property 'replace' of undefined
```

The line in error is:

```
this.editor.replace(this.replaceInput.value);
```

It seems that `this.editor` is undefined.

Chrome Browser Version 41.0.2272.76 (64-bit) on Mac OS X 10.10

I am willing to make PR. First of all I need to get it up and running. I think I have something wrongly setup otherwise the `Replace` should work.

**Comment by coderaiser** [comment]
Fixed in [v1.0.6](https://github.com/coderaiser/cm-searchbox/releases/tag/v1.0.6).
Thank you.
